### DISCUSSION TITLE
Is it possible to apply theme values to inline/direct styles?

### DISCUSSION THREAD

**Discussion by mohandeblaze** [chakra-ui/chakra-ui#3632] [Q&A]
Hi, I have simple button with some custom styling like below

```tsx
export function TransparentButton(props: ButtonProps) {
    const theme = useTheme();

    return (
        <RegularButton
            css={css({ border: '1px solid green.500' })(theme)}
            background="transparent"
            color="color.level-900"
            {...props}
        />
    );
}
```

![image](https://user-images.githubusercontent.com/11157953/111901584-1db72480-8a5f-11eb-862d-224f79bbeea9.png)

I'm wondering is it possible to apply theme value into direct css style like `css={css({ border: '1px solid green.500' })(theme)}`. Because it always generates the CSS like in the image. Am i doing anything wrong in here or is it not supported at all?

**Comment by malashevskyi** [comment]
Seems it is impossible.
I tried to configure it with webpack, it is available with postcss plugin but obviously only if vars defined in css, not in js like in chakra: 
![screenshot](https://i.imgur.com/31d43Ld.png)
But you can see styles in the 'Computed' tab (or press button to see in the right side), it is not as comfortable as 'Styles' tab but... (it is not an answer but maybe help someone)
![screenshot](https://i.imgur.com/h9Zbwsk.png)
